readme.md

```diff
@@ -9,44 +9,67 @@ Hosted version is available at https://yearn.vision.
 
 You will need:
 
-- Erigon for querying historical data
-- Victoria-metrics to pull the metrics, persist them and make them queryable
-- Grafana if you want to set up custom dashboards and alerts
-- Etherscan API key
-- docker and docker-compose (not mandatory but easier usage, see below)
+- Etherscan API key (and API keys for other networks that you want to use)
+- [Docker](https://www.docker.com/) and [Docker Compose](https://github.com/docker/compose
 
 ## Usage
 
-### Prometheus exporter
+Run `make up` to start the exporter.
```

Comment:
to start all of the exporters*

we should prbably include mentions of the individual exporter commands in case the user only needs a subset of data

Reply:
```suggestion
Run `make up` to start all of the exporters.
```

Reply:
ah nvm, I see you did that below. Maybe just a blurb saying that the functionality exists and to look below for info